.github/actions/check-pr-format/action.yml

```diff
@@ -0,0 +1,35 @@
+name: PR Title Check
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR title
+        run: |
+          const title = context.payload.pull_request.title;
+          const regex = /(Add|Remove|Update|Fix)\s*.+/;
+          const regexNoSpace = /\[(Add|Remove|Update|Fix)\].+/i;
+          const regexLowerCase = /\[(add|remove|update|fix)\]\s*.+/i;
+          const issue_number = context.payload.pull_request.number;
+          const owner = context.repo.owner;
+          const repo = context.repo.repo;
+          let body = '';
+
+          if (!regex.test(title)) {
+            if (regexNoSpace.test(title)) {
+              body = 'Your PR title is missing a space after the keyword. Please update it to: [Add | Remove | Update | Fix] <Title>. For example: Remove Old feature';
+            } else if (regexLowerCase.test(title)) {
+              body = 'The keyword in your PR title is not capitalized. Please update it to: [Add | Remove | Update | Fix] <Title>. For example: Update Existing feature';
+            } else {
+              body = 'Your PR does not have a [Add | Remove | Update | Fix] <Title>. Please add one to help streamline our changelog. For example: Fix Bug in feature';
+            }
+            github.issues.createComment({ issue_number, owner, repo, body });
```

Comment:
In addition to adding a comment can we also fail this action? If the action still succeeds we probably can't count on a comment enforcing this behavior.

Reply:
I am of the philosophy to warn first and move to blocking if warning doesn't yield results. Several engineers also expressed concern at blocks causing friction, we can always add blocking in the near future if the comments alone don't work.

Reply:
If we're not going to block on the title, this is a recommendation and not a requirement. If we want consistency in the PR titles, this should be a requirement. If it's a requirement, what's the ask for reviewers? Ignore the title and approve anyway? If it's anything else, where the reviewer asks the contributor to update the title, then this needs to be a requirement.

Reply:
I agree with making it a requirement (can be in a quick follow up PR if we want to test this first). I'd also prefer if we expand the keywords to allow verb<ing> (adding/removing) which is very commonly used by many developers. 

Unrelated to this PR - can this be used to enforce on commit messages as well in addition to PR titles?